documentation/assemblies/overview/assembly-metrics-overview.adoc

```diff
@@ -0,0 +1,37 @@
+// This assembly is included in:
+//
+// overview/master.adoc
+
+[id="metrics-overview_{context}"]
+= Monitoring
+
+Metrics data allows you to monitor the performance and health of {ProductName}.
```

Comment:
The introductory paragraph talks about metrics data while it should tak about monitoring data since it includes for example also Jaeger.

```suggestion
Monitoring data allows you to monitor the performance and health of {ProductName}.
```